CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 #
 # Use git ls-files '<pattern>' without a / prefix to see the list of matching files.
 
+/.ci.yaml @caseyhillers
```

Comment:
Can we add a group rather than a person? if not possible can we add at least one more person?

Reply:
cc @kf6gpe regarding creating a team under the Flutter github org

Reply:
For now, I added Chris since he'll be working on the single recipe builds in this file.

While we're at it, should we create subteams for the other teams as well?

Reply:
I think you can put them in the same line.
```yaml
/.ci.yaml @caseyhillers @christopherfujino
```